examples/metro.config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+const path = require("path");
+const blacklist = require("metro-config/src/defaults/blacklist");
+
+const reactNativeSplachScreenRoot = path.resolve(__dirname, "..");
+
+module.exports = {
+  projectRoot: path.resolve(__dirname, "."),
+
+  transformer: {
+    getTransformOptions: async () => ({
+      transform: {
+        experimentalImportSupport: false,
+        inlineRequires: false
+      }
+    })
+  },
+  watchFolders: [
+    path.resolve(__dirname, "node_modules"),
+    reactNativeSplachScreenRoot
```

Comment:
nice

Reply:
Typo: where it is _Splach_ should be _Splash_